ee/cis/macos-14/README.md

```diff
@@ -0,0 +1,37 @@
+# macOS 14.0 Sonoma benchmark
+
+Fleet's policies have been written against vTODO of the benchmark. You can refer to the [CIS website](https://www.cisecurity.org/cis-benchmarks) for full details about this version.
```

Comment:
```suggestion
Fleet's policies have been written against v1.0.0 of the benchmark. You can refer to the [CIS website](https://www.cisecurity.org/cis-benchmarks) for full details about this version.
```

We cover 1.0.0. @sharon-fdm please correct me if I'm wrong.

Here's where we're tracking updates to CIS Benchmark versions: https://docs.google.com/spreadsheets/d/1IWfQtSkOQgm_JIQZ0i2y3A8aaK5vQW1ayWRk6-4FOp0/edit#gid=0

Reply:
@noahtalerman 


[Versions](https://docs.google.com/spreadsheets/d/1HZn7e4zpd5nfxY0MhSdX8MkGpdpxyLdSZjk0Auo1WaQ/edit#gid=1561077273)

macos 13 [1.0.0](https://drive.google.com/file/d/1VL8e2wmApkayv-9gM8zlyYljP6phsQhO/view?usp=share_link)
macos 14[1.0.0](https://drive.google.com/file/d/10hSWBZ06_XTwxC0GNjsjwObcQk72NQi8/view)
Win10 [2.0.0](https://drive.google.com/file/d/1agln3JBRkwJcefHibXzLMai31UuvTsDm/view)
Win11 [2.0.0](https://drive.google.com/file/d/1Ol_TKqB96M4R4QCG7vkwCbWVV6fpZZnK/view?usp=drive_link)